inspire/modules/deposit/static/js/deposit/literature_submission_form.js

```diff
@@ -780,7 +784,31 @@ define(function(require, exports, module) {
         url: that.save_url,
         form_selector: that.$submissionForm
       });
-    }
+    },
+
+    isFormBlank: function isFormBlank() {
+      var $inputFields = $('input').not('[id$="__last_index__"]');
+      $inputFields = $inputFields.add($('textarea'));
+      var isBlank = true;
+
+      $.each($inputFields, function() {
+        var type = this.type;
+        if (type === 'text' || type === 'password' || type === 'hidden' || type === 'textarea') {
+          if(this.value !== ''){
+            isBlank = false;
+            return false;
+          }
+        } else if ((type === 'checkbox' || type === 'radio') && this.checked) {
+          isBlank = false;
+          return false;
+        } else if (type === 'select-multiple' && $('#subject').val() !== null) {
+          isBlank = false;
+          return false;
+        }
+      });
+
+      return isBlank;
```

Comment:
The `return false` is just to break out of the `$.each()` so it doesn't check more fields, whereas the `isBlank` is changed accordingly and by default is set to `true`.